www/content/tutorial.md

```diff
@@ -440,6 +440,12 @@ You can give `dbg` any expression you like, for example:
 dbg Str.concat singular plural
 ```
 
+You can also use `dbg` inside an expression: it will print its input to stderr and return it to the caller. For example:
```

Comment:
Use of the indefinite article here is a bit imprecise 
```
it will print its input to stderr and return it to the caller
^^ dbg        ^^^ dbg                        ^^ input
```
What about
```suggestion
You can also use `dbg` as a function inside an expression, which will print the function argument to stderr and then return the argument to the caller. For example:
```

Reply:
`dbg` isn't actually a function, but it has similar parsing rules when you use it as an expression so I think "as a function" is helpful. For example the tutorial uses the example `dbg Str.concat singular plural`, but in expression position that would have to be `dbg (Str.concat singular plural)`.